values.schema.json

```diff
@@ -187,6 +190,7 @@
         "path": { "type": "string", "title": "ingress path" },
         "pathType": { "type": "string", "title": "ingress pathType", "default": "ImplementationSpecific" },
         "tls": { "$ref": "#/definitions/tls" },
+        "className": { "$ref": "#/definitions/className" },
```

Comment:
It is not reusable field, just declare schema inline.

Reply:
Fixed, plus added tests. Btw, there are no makefiles [as it says](https://github.com/eshepelyuk/cmak-operator/blob/master/CONTRIBUTING.md?plain=1#L27) in  contributing page.

Reply:
Thnx will fix it